Foundation/NSTask.swift

```diff
@@ -7,6 +7,7 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
+#if !os(Android)
```

Comment:
I don't want to comment out whole files for specific platforms like this. Is it impossible to support? If so we should put a fatalError in a useful place (like init).

Either that or we need a better availability strategy for the API (e.g., marking the class as unavailable on android only).